README.md

```diff
@@ -1,288 +1,112 @@
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
+- [📗 Table of Contents](#-table-of-contents)
+- [Recipes App](#recipes-app)
+  - [Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Available Scripts](#available-scripts)
+  - [Authors](#authors)
+  - [Future Feature](#future-feature)
+  - [Contributing](#contributing)
+  - [Show your support ](#show-your-support-)
+  - [Acknowledgments](#acknowledgments)
+  - [License](#license)
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
+# Recipes App
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
 
-<!-- Features -->
+**Recipes App** is web app that allow users to add food and create recipes, the app is built with Ruby on Rails and has a user authentication and an authorization system.
 
-### Key Features <a name="key-features"></a>
+## Built With
 
-> Describe between 1-3 key features of the application.
+### Tech Stack 
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+<ul>
+  <li><a href="https://www.ruby-lang.org/en/news/2022/11/24/ruby-3-1-3-released/">Ruby 3.1</a></li>
+  <li><a href="https://rubyonrails.org/">Ruby on Rails 7</a></li>
+  <li><a href="https://www.postgresql.org/">PostgreSQL 15</a></li>
+</ul>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Key Features
 
-<!-- LIVE DEMO -->
+- User registration system (using Devise gem)
```

Comment:
Thank you for your feedback. Since we are running short of the deadline so we would skip this suggestion for now. But surely we will revisit and consider the suggestion as we believe it would greatly improve our project.